src/include/daos_srv/control.h

```diff
@@ -33,7 +33,17 @@ dpdk_cli_override_opts;
 #define NVME_DEV_STATE_NORMAL	(NVME_DEV_FL_PLUGGED | NVME_DEV_FL_INUSE)
 #define NVME_DEV_STATE_FAULTY	(NVME_DEV_STATE_NORMAL | NVME_DEV_FL_FAULTY)
 #define NVME_DEV_STATE_NEW	NVME_DEV_FL_PLUGGED
-#define NVME_DEV_STATE_INVALID	(1 << 4)
+#define NVME_DEV_STATE_INVALID	(1 << 4) /* Unsupported device or LED state */
```

Comment:
as said elsewhere, now we have LED_STATE, LED references should be removed from DEV_STATE